### ISSUE TITLE
Inline ads and video player

### ISSUE THREAD

**Issue by tcptomato** [uBlockOrigin/uAssets#97]
### URL(s) where the issue occurs

http://www.trafictube.ro/depasire-ratata-faci-depasire/

### Describe the issue

Ad generating script and video player were both inlined, so that blocking inline scripts doesn't work. Cosmetic filter (like ###primary > .smecher - for the one above the player ) doesn't help, ads are just being reloaded.

### Screenshot(s)

[Screenshot(s) for difficult to describe visual issues are **mandatory**]

### Versions

- Browser/version: Chrome 52
- uBlock Origin version: 1.8.4

### Settings

added my own custom list.

### Notes

Divs are generated with random IDs by some inline script. Blocking inline scripts on the page also kills the loading of the video players needed. Adding the video player code ( at the bottom in the source of the linked page) to the resource library to that they can be injected into the page might  fix this issue.

**Comment by gorhill** [comment]
trafictube.ro##.smecher:style(height: 1px !important;)
    trafictube.ro##.widget-smecher:style(height: 1px !important;)

I did not see video ads.

**Comment by tcptomato** [comment]
Thanks, but this is just a temporary fix. 

The main problem is that in order to use the site ( the youtube player to load), I can't block inline scripting on the page, and this allows them to run whichever inline script they like ( and they'll start checking for div size). This was why in my opinion injecting the needed code through the resource library would have been a good approach. 

I can see the reasons you wouldn't want this approach, I just wanted to have this discussion with you.

**Comment by tcptomato** [comment]
Thanks ...

**Comment by gorhill** [comment]
> Thanks ...

Here is how I interpret these three dots:

> Your solution works but I wish you spent more time to provide me with a solution which satisfies me. Despite that your solution works, that you refuse to get back to work to for a solution which meets my specific requirement is really disappointing.

Remember: it's all volunteer-based work.

**Comment by tcptomato** [comment]
Sorry, it was more venting frustration at you missing my point. But let me share how I've read your replies 

>  Here is a solution to the current issue now leave me alone.

I didn't want for you to take my place in the whack-a-mole game I'm playing with that site, I wanted to know your opinion on that approach, to know what options I'll have when the escalation will reach the point when using the resource library would be the only viable option.

**Comment by gorhill** [comment]
Code-injection solution is always much more time-consuming than those new cosmetic filter options, this is one of the main reason I decided to implement these new options, to save time, that is why I rather use them if they work. Either ways, a site can always work around whatever we come up with, _including_ injected code. An example of such is Yavli's ads, which code I constantly need to update every few days. I could make the code more complicated to make their job more difficult, but they will do the same on their side (I tried once, they did, I reverted to the simpler code). If the site here keep working around what we come up with, I will then be willing to look for code -- so long as the site's amount of visits justifies the work.
